src/mumble/AudioInput.cpp

```diff
@@ -814,6 +816,39 @@ bool AudioInput::selectCodec() {
 	return true;
 }
 
+void AudioInput::selectNoiseCancel() {
+
+	//noiseCancel = g.s.noiseCancel;
```

Comment:
Unnecessary comment?

Reply:
That comment will replace the line below once the UI code change is done and 	`bool bDenoise;` will be replaced with `NoiseCancel noiseCancel;` in `Struct settings`.